.github/workflows/build.yml

```diff
@@ -10,14 +10,14 @@ on:
 
 jobs:
   clang-format-check:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     name: Clang-Format Check
     steps:
       - uses: actions/checkout@v4
-      - uses: RafikFarhad/clang-format-github-action@v4
+      - uses: jidicula/clang-format-action@v4.13.0
```

Comment:
Is there not an official one you can use? Either way, looks good, thanks for catching this

Reply:
This is the more popular I could find. Happy to be proven wrong, though!